# based on: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=905384
# based on: Description: Correct the libMUSCLE.la dependency
# based on: This allows automake to generate proper dependencies.
# based on: Author: Adrian Bunk <bunk@debian.org>
# SEP 17th 2020 by B. Hajgato (UGent)

--- libmuscle-3.7+4565.orig/libMUSCLE/Makefile.am
+++ libmuscle-3.7+4565/libMUSCLE/Makefile.am
@@ -36,6 +36,6 @@ bin_PROGRAMS = muscle
 
 
 muscle_SOURCES = main.cpp
-muscle_LDADD = $(top_builddir)/libMUSCLE/libMUSCLE-3.7.la
+muscle_LDADD = libMUSCLE-3.7.la
 
 
